setup.py

```diff
@@ -48,7 +48,9 @@ def prerelease_local_scheme(version):
         'girder_client>=2.3.0',
         'jinja2',
         'jsonpath-rw',
-        'avogadro'
+        'avogadro',
+        'IPython',
+        'ipykernel'
```

Comment:
I think we just want to do guarded imports rather than making this a hard requirement.